action.yml

```diff
@@ -29,13 +29,25 @@ inputs:
 runs:
   using: 'composite'
   steps:
+
+    - name: Getting previous run state
```

Comment:
What will happen if we miss previous run state?

Reply:
good point, I forgot to add continue-on-error. Fixed